Documentation/design-docs/COM-activation.md

```diff
@@ -0,0 +1,92 @@
+# COM Activation for .NET Core on Windows
+
+## Purpose
+
+In order to more fully support the vast number of existing .NET Framework in their transition to .NET Core, support of the COM Activation scenario in .NET Core is required. Without this support it is not possible for many .NET Framework consumers to even consider transitioning to .NET Core. The intent of this document is to describe aspects of the activation of a .NET class written for .NET Core within a COM environment. This support includes but is not limited to activation via the [`CoCreateInstance()`](https://docs.microsoft.com/en-us/windows/desktop/api/combaseapi/nf-combaseapi-cocreateinstance)API in C/C++, from a [Windows Script Host](https://docs.microsoft.com/en-us/windows/desktop/com/using-com-objects-in-windows-script-host), or as a [Office VSTO Add-In](https://docs.microsoft.com/en-us/visualstudio/vsto/create-vsto-add-ins-for-office-by-using-visual-studio).
+
```

Comment:
`Office VSTO Add-In` comes with number of requirements. We have said earlier that we are not going to worry about it for .NET Core 3.0.

Reply:
Confusion on my part. Will remove.